parser/Builder.cc

```diff
@@ -558,6 +596,11 @@ class Builder::Impl {
         return make_unique<Const>(tokLoc(name), nullptr, gs_.enterNameConstant(name->string()));
     }
 
+    unique_ptr<Node> const_pattern(unique_ptr<Node> const_, const token *begin, unique_ptr<Node> pattern,
```

Comment:
I believe most of the methods in this file are `camelCase`. Would you mind following that convention?

Reply:
Before my changes:

38 snake_case: ``` array_pattern,
    call_method,
    case_match,
    const_pattern,
    def_class,
    def_sclass,
    hash_pattern,
    if_guard,
    in_match,
    in_pattern,
    line_literal,
    loop_while,
    loop_while_mod,
    multi_assign,
    multi_lhs,
    multi_lhs1,
    not_op,
    nth_ref,
    op_assign,
    p_ident,
    pair_keyword,
    pair_quoted,
    range_exclusive,
    range_inclusive,
    rational_complex,
    regexp_compose,
    regexp_options,
    rescue_body,
    splat_mlhs,
    string_compose,
    string_internal,
    symbol_compose,
    symbol_internal,
    symbols_compose,
    unary_op,
    unless_guard,
    words_compose,
    xstring_compose ```

28 CamelCase: ```attrAsgn,
    backRef,
    beginBody,
    beginKeyword,
    binaryOp,
    blockPass,
    callLambda,
    conditionMod,
    constFetch,
    constGlobal,
    constOpAssignable,
    dedentString,
    fileLiteral,
    floatComplex,
    indexAsgn,
    keywordBreak,
    keywordDefined,
    keywordNext,
    keywordRedo,
    keywordRetry,
    keywordReturn,
    keywordSuper,
    keywordYield,
    keywordZsuper,
    logicalAnd,
    logicalOr,
    loopUntil,
    loopUntil_mod```

I went with the most used.

Reply:
makes sense, thanks!